api/projectTeam.js

```diff
@@ -23,13 +27,13 @@ router.get("/self", async (req, res) => {
 router.put("/join/:name", async (req, res) => {
   try {
     const projectTeam = await models.ProjectTeam.findOne({
-      where: { name: req.params.name },
-      include: [models.Person]
+      where: { name: req.params.name }
     });
-    if (projectTeam.People.length >= 4) {
+    if (projectTeam.Members.length >= 4) {
       throw new Error("Team is full");
     }
-    projectTeam.addPerson(req.user.id);
+    await projectTeam.addMember(req.user.id);
+    await projectTeam.reload();
```

Comment:
does `addMember` not already in-place update the projectTeam?

Reply:
It wasn't not sure why